actors/abi/bitfield.go

```diff
@@ -110,6 +110,16 @@ func (bf BitField) Set(bit uint64) {
 	bf.bits[bit] = struct{}{}
 }
 
+// Unset removes bit from the BitField
+func (bf BitField) Unset(bit uint64) {
+	delete(bf.bits, bit)
+}
+
+// Has returns true iff bif is set in the BitField.
+func (bf BitField) Has(bit uint64) bool {
+	return bf.bits[bit] == struct{}{}
+}
+
```

Comment:
These didn't exist in the lotus implementation, added them here to fit our usage.

Reply:
I'm pretty sure these won't just work a implemented here, cc @Kubuxu